### PR TITLE
add target to links and add fallback text for repos without description

### PR DESCRIPTION
**Pull request by andrewchar**
solves Issue #11

### CODE REVIEW

File: src/components/UserEventsList/UserEventsList.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ class UserEventsList extends Component {
     }
 
     handleFilterChange(type, event) {
-        console.log("Pupu");
+        console.log("Pupu"); /*<- lol wut*/
```
Comment (razat249) [inline src/components/UserEventsList/UserEventsList.jsx]: Just some funny names for debugging :smile_cat:

File: src/components/UserEventsList/UserEventsList.jsx [hunk 2]
```diff
@@ -101,25 +101,29 @@ class UserEventsList extends Component {
...
+                        eventData.title = (<p><b>Made a branch
+                                            <a href={ github_base_url + event.repo.name + '/tree/' + event.payload.ref } target="_"> { event.payload.ref } </a>
                                             from
                                             <a href={ github_base_url + event.repo.name } target="_"> {event.repo.name}</a>
                                           </b></p>);
                     }
-                    eventData.data = <p><b>Repo description:</b> { event.payload.description }</p>
-                    eventData.iconColor = "orange";
+                    if (event.payload.description === null) {
+                      eventData.data = <p><b>Repo description:</b> No description provided.</p>
```
Comment (razat249) [inline src/components/UserEventsList/UserEventsList.jsx]: Nice fix 👍

File: src/components/UserEventsList/UserEventsList.jsx [hunk 3]
```diff
@@ -101,25 +101,29 @@ class UserEventsList extends Component {
...
+                    } else {
+                      eventData.data = <p><b>Repo description:</b> { event.payload.description }</p>
+                      eventData.iconColor = "orange";
```
Comment (razat249) [inline src/components/UserEventsList/UserEventsList.jsx]: I think iconColor should be out of this else block.

**Review by razat249 — CHANGES_REQUESTED** [review]
Noce work! A little changea required.

**Comment by andrewchar** [comment]
requested changes are complete.

**Review by razat249 — APPROVED** [review]
Nice, keep up the good work :)

**Pull request merged**